queries.sql

```diff
@@ -1,10 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * FROM animals WHERE name LIKE '%mon';
-SELECT * FROM animals WHERE EXTRACT(YEAR FROM date_of_birth) BETWEEN 2016 AND 2019;
-SELECT * FROM animals WHERE neutered = TRUE AND escape_attempts < 3;
-SELECT * FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
-SELECT * FROM animals WHERE weight_kg > 10.5;
-SELECT * FROM animals WHERE neutered = TRUE;
-SELECT * FROM animals WHERE NOT (name = 'Gabumon');
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT name FROM animals WHERE name LIKE '%mon';
+SELECT date_of_birth FROM animals WHERE EXTRACT(YEAR FROM date_of_birth) BETWEEN 2016 AND 2019;
+SELECT neutered, escape_attempts FROM animals WHERE neutered = TRUE AND escape_attempts < 3;
```

Comment:
- [ ] Same here as we can see in the screenshot of the project requirements below
![Screenshot from 2023-05-02 08-10-52](https://user-images.githubusercontent.com/46388113/235612663-329cfe96-287c-46a8-b8f0-2fca4582d8f0.png)